### PR TITLE
Telemedicine flag

### LINKED ISSUE #962: Consultation Update filed by an external specialist: Telemedicine
We have a telemedicine module inside care.
the reason for having this module is here:

There may be patients in small healthcare facilities with no specialist doctors.
When these patients become severely sick, There may not be enough resources to shift the patient to a hospital with specialist doctors.
Instead, the patient remains in the small healthcare facility, and he is put under telemedicine with a specialist doctor sitting elsewhere.
This is so that the patient is admitted in a facility and gets around the clock medical attention while He also gets the advanced care of a specialist through the telemedicine module of CARE.

The feature is explained as under:
Patient A is admitted in facility X. While the facility files consultation for the patient, telemedicine is enabled, and a Doctor P is assigned.
Now when doctor P logs into CARE, in the patients' tab, he can view the card and consultations of the patient.

**Is your feature request related to a problem? Please describe.**
Once doctor P views the patients clinical details, he needs to record his consultation in a consultation update. 
The consultation update filed by an external expert must be prominently flagged as an external consultation update. Otherwise, the facility X may find it confusing. it will be hard for them to distinguish theirs and experts consultation update. 

**Describe the solution you'd like**
The consultation update filed by the external specialist must appear in a different colour and must show who the consultation update is from. 
The consultation card on the patient dashboard must also have a flag that it contains a consultation update from an external expert.

### PR DESCRIPTION
**Pull request by sam9111**
UI change: Color changes for consultation updates that are done by the external specialist,the doctor's name is displayed in that particular update and a badge appears in the patient home. Resolves #962

### CODE REVIEW

File: src/Components/Common/PrescriptionBuilder.gen.tsx [hunk 1]
```diff
@@ -4,8 +4,9 @@
 
 import * as React from 'react';
 
-// tslint:disable-next-line:no-var-requires
-const PrescriptionBuilderBS = require('./PrescriptionBuilder.bs');
+// @ts-ignore: Implicit any on import
+import * as PrescriptionBuilderBS__Es6Import from './PrescriptionBuilder.bs';
```
Comment (bodhish) [inline src/Components/Common/PrescriptionBuilder.gen.tsx]: Can you revert these changes

**Comment by sam9111** [comment]
Added a button to the consultation update details page as well resolving issue #1128

**Review by bodhish — COMMENTED** [review]
LGTM
